examples/lighting-app/linux/README.md

```diff
@@ -40,7 +40,7 @@ Raspberry Pi Desktop 20.10 (aarch64)**
 
 > If you want to test ZCL, please disable Rendezvous
 >
->     gn gen out/debug --args='bypass_rendezvous=true'
+>     gn gen out/debug --args='chip_bypass_rendezvous=true'
 >     ninja -C out/debug
 >
 > Note that GN will set bypass_rendezvous for future builds, to enable
```

Comment:
`bypass_rendezvous` here needs to be changed to `chip_bypass_rendezvous` as well?